src/cli/dsc/schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Oh My Posh DSC State",
+  "description": "State configuration for Oh My Posh DSC (Desired State Configuration)",
+  "properties": {
+    "configurations": {
+      "type": "array",
+      "title": "Configuration List",
+      "description": "List of Oh My Posh configurations",
+      "items": {
+        "type": "object",
+        "properties": {
+          "path": {
+            "type": "string",
+            "title": "Configuration Path",
+            "description": "File path to the Oh My Posh configuration file"
+          },
+          "format": {
+            "type": "string",
+            "title": "Configuration Format",
+            "description": "Format of the configuration file",
+            "enum": [
+              "json",
+              "yaml",
+              "toml"
+            ]
```

Comment:
Can this format mismatch with the `path`? How does it behave if it does?

Reply:
It could, and then we end up in a broken state. Which can only happen then someone manually crafts a state though. I was thinking about just using the path's extension and not make this explicit.

Reply:
You could definitely just use the path extension and add a validation pattern to ensure it ends in a supported format extension.